config/detekt-baseline.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <SmellBaseline>
   <Blacklist timestamp="1521663963552"></Blacklist>
-  <Whitelist timestamp="1522100300281">
+  <Whitelist timestamp="1522877314667">
```

Comment:
Let's try to avoid updating the baseline. Either let's fix the issues or use the @Suppress annotation in code. Otherwise this list gets longer and longer and we can never get rid of it.